turbo/src/lib/scripts/build_nodes.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
```

Comment:
Even though it's a script, I would still make it a method with documentation with a `if __name__ == "main":`
The documentation would be welcome, even though it's pretty easy to understand, it still takes cognitive effort without doc.